<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatDialog</class>
 <widget class="QDialog" name="StatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据统计</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>个人统计</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,10">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="8,9">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>请输入统计人的编号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_11"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="6,2,1,2,1">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_121">
           <property name="text">
            <string>请输入统计的时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_3">
           <property name="minimum">
            <number>2001</number>
           </property>
           <property name="maximum">
            <number>2045</number>
           </property>
           <property name="value">
            <number>2017</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_122">
           <property name="text">
            <string>年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_4">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_123">
           <property name="text">
            <string>月</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="pushButton_1">
         <property name="maximumSize">
          <size>
           <width>338</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>开始统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_1"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>部门统计</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,10">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="8,9">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>请选择统计部门的名称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>保安部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>销售部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>宣传部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>人事部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>财务部 </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>技术部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>仓储部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>研发部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>维修部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>行政部</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="6,2,1,2,1">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_124">
           <property name="text">
            <string>请输入统计的时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="minimum">
            <number>2001</number>
           </property>
           <property name="maximum">
            <number>2045</number>
           </property>
           <property name="value">
            <number>2017</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_125">
           <property name="text">
            <string>年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_126">
           <property name="text">
            <string>月</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>开始统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>企业统计</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,10">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="6,2,1,2,1">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_127">
           <property name="text">
            <string>请输入统计的时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_5">
           <property name="minimum">
            <number>2001</number>
           </property>
           <property name="maximum">
            <number>2045</number>
           </property>
           <property name="value">
            <number>2017</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_128">
           <property name="text">
            <string>年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_6">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_129">
           <property name="text">
            <string>月</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>开始统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>9</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>暂无数据！</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
